index.js

```diff
@@ -26,6 +27,37 @@ server.listen(port, ipaddress, () => {
 
 const mailbox_storage = JSON.parse(fs.readFileSync("./mailbox.json", "utf8"));
 
+function updateNumbers(){
+	fs.writeFileSync("./json/numbers.json", JSON.stringify(numbers), "utf8");
+	numbers = JSON.stringify(numbers);
+}
+
+function removeNumber(numberIndex){
+	numbers.splice(numberIndex, 1);
+}
+
+schedule.scheduleJob({date: 1, hour: 0, minute: 0, second: 0}, function(){
+	const now = new Date();
+	for (var i in numbers){
+		const number = numbers[i];
+		if (number.year <= now.getFullYear() || number.month <= now.getMonth()){
+			if (number.month == now.getMonth() || (number.month == 12 && now.getMonth() == 0)){
+				// send a notice to the user.
```

Comment:
This is not how you get channels ever. You use get(ID), where id is number.channel in this case